### PR TITLE
Cran summary wait

### PR DESCRIPTION
**Pull request by maelle**
Good point @gaborcsardi!

### CODE REVIEW

File: R/check-class.R [hunk 1]
```diff
@@ -227,6 +227,12 @@ check_cran_summary <- function(self, private) {
 
   x <- private$status_
 
+  if (any(vapply(x, function(xx) xx$status, "") %in% 
+     c("in-progress", "created"))) {
+    stop("At least one of the builds has not finished yet. Please wait before calling cran_summary() again.",
```
Comment (gaborcsardi) [inline R/check-class.R]: can you pls break the long line?

Comment (gaborcsardi) [reply R/check-class.R]: Also, can you use `map_chr()` Thanks

Comment (gaborcsardi) [reply R/check-class.R]: and quote the function name in backticks? Thanks.

Comment (maelle) [reply R/check-class.R]: ok!

**Comment by maelle** [comment]
What do you think should happen if one of the builds has a preperror or so?

**Comment by gaborcsardi** [comment]
Should be a separate category in output, failed to check, and also it should trigger the message.

**Comment by maelle** [comment]
which message?

**Comment by gaborcsardi** [comment]
The message you just added.

**Comment by maelle** [comment]
the one about fixing all problems? ok. I'll get to this next week.

**Comment by gaborcsardi** [comment]
Thanks!

**Pull request merged**